### ISSUE TITLE
Bug: Organize the Header

### ISSUE THREAD

**Issue by ayushsinha25** [OWASP/BLT#1852]
Removing the extra gap between elements in header
![Screenshot 2024-02-22 161421](https://github.com/OWASP/BLT/assets/94716274/9295ca91-73fc-4910-920f-aeb4a6b43ab6)

**Comment by AtmegaBuzz** [comment]
Did you discussed this change with @DonnieBLT before reporting it @ayushsinha25 ?

**Comment by ayushsinha25** [comment]
> Did you discussed this change with @DonnieBLT before reporting it @ayushsinha25 ?

Yep, but it was closed due to linting process, and he asked me to make the PR again after the process was over

**Comment by AtmegaBuzz** [comment]
Understood
